### PR TITLE
[-] Upgrade NBX kernels

### PR DESCRIPTION
**Pull request by cdevent**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

A batch of fixes and improvements for the NBX drop-ins

### CODE REVIEW

**Review by c-h-russell-walker — APPROVED** [review]
Nice, LGTM 👍

**Comment by cdevent** [comment]
jenkins run test_integration_general

**Comment by cdevent** [comment]
jenkins run test_integration_python_keras

**Comment by cdevent** [comment]
jenkins run test_integration_general

File: public_dropin_notebook_environments/python38_snowflake_notebook/requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 altair==4.2.0
 dask==2022.9.2
 datarobot-bp-workshop==0.2.6
-datarobot-drum==1.10.3
+datarobot-drum==1.10.4
```
Comment (yakov-g) [inline public_dropin_notebook_environments/python38_snowflake_notebook/requirements.txt:4]: please change to 1.10.5

File: public_dropin_notebook_environments/python39_notebook/requirements.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 altair==4.2.0
 dask==2022.9.2
 datarobot-bp-workshop==0.2.6
-datarobot-drum==1.10.3
+datarobot-drum==1.10.4
```
Comment (yakov-g) [inline public_dropin_notebook_environments/python39_notebook/requirements.txt:4]: And here

**Review by yakov-g — APPROVED** [review]
Just change DRUM version

**Comment by c-h-russell-walker** [comment]
> Just change DRUM version

I also made a PR for this update in NBX code here:
https://github.com/datarobot/notebooks/pull/2038

**Comment by yakov-g** [comment]
jenkins run test_inference_model_templates

**Pull request merged**